packages/react-components/src/localization/locales/de-DE/strings.json

```diff
@@ -216,4 +216,4 @@
     "step3DigitText": "3",
     "step4DigitText": "4"
   }
-}
\ No newline at end of file
+}

```

Comment:
This happened magically when I was merging back from main branch, is it prettier or some other things doing their job?

Reply:
Unfortunate effect of merging string translations into `main` from another PR.
Wait for some other PR to take the brunt of fixing this when you merge from `main` again :P